scripts/makeLegacyPlots.sh

```diff
@@ -77,83 +79,69 @@ echo $obj1,$obj2
 # and an option to unblind only part of the DNN score spectrum:
 #     1. --blind-range -> do not plot the data in teh specified range 
 #                      -> takes as input two numbers: MIN and MAX of range to be blinded
-
-
+# TEST OPTIONS
+#python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel2 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.6 1 --no-binwidth --binNXaxis --doStatSystBand --addJERunc
+#python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel2 --channel $channel --tag $tag --label "DNN score"      --quit --ratio --log --blind-range 0.6 1 --binNXaxis --doStatSystBand --addJERunc
+#python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel2 --channel $channel --tag $tag --label "DNN score"      --quit --ratio --log --blind-range 0.6 1 --doStatSystBand --addJERunc
+#python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel5 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.6 1 --no-binwidth --binNXaxis
+#python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel5 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.6 1 --no-binwidth 
+python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel5 --channel $channel --tag $tag --label "DNN score" --quit --ratio --log --blind-range 0.6 1 --binNXaxis 
+python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel5 --channel $channel --tag $tag --label "DNN score" --quit --ratio --log --blind-range 0.6 1 
+#python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel5 --channel $channel --tag $tag --label "DNN score" --quit --ratio --log --blind-range 0.6 1 --binNXaxis --doStatSystBand --addJERunc
+#python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel5 --channel $channel --tag $tag --label "DNN score" --quit --ratio --log --blind-range 0.6 1 --doStatSystBand --addJERunc
+#python scripts/$plotter --dir $basedir/analysis\_$tag/$channel\_$year/total --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel5 --channel $channel --tag $tag --label "DNN score"      --quit --ratio --log --blind-range 0.6 1 --doStatSystBand --addJERunc
```

Comment:
Please remove these and instead put:
- 8 lines for the final plot as function of DNN score
- 8 lines for the final plot as function of DNN bin number

Reply:
The 16 un-commented lines requested here are missing.